### PR TITLE
chore(create-instantsearch-app): update links to monorepo

### PR DESCRIPTION
**Pull request by FabienMotte**
**Summary**

This PR updates the links pointing to the monorepo for the `create-instantsearch-app` package.

**Result**

Links to the old `algolia/instantsearch.js` repo are now pointing to the `algolia/instantsearch` monorepo.
e2e test snapshots have been updated as well.

### CODE REVIEW

File: packages/create-instantsearch-app/README.md [hunk 1]
```diff
@@ -147,14 +150,14 @@ app.create().then(() => console.log('App generated!'));
...
-- [InstantSearch.js 2](https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/instantsearch.js-2.x)
-- [React InstantSearch](https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/react-instantsearch)
-- [Vue InstantSearch](https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/vue-instantsearch)
-- [Angular InstantSearch](https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/angular-instantsearch)
-- [Autocomplete.js](https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/autocomplete.js)
-- [JavaScript Client](https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/javascript-client)
-- [JavaScript Helper](https://codesandbox.io/s/github/algolia/instantsearch.js/tree/templates/javascript-helper)
+- [InstantSearch.js 3](https://codesandbox.io/s/github/algolia/instantsearch/tree/templates/instantsearch.js)
+- [InstantSearch.js 2](https://codesandbox.io/s/github/algolia/instantsearch/tree/templates/instantsearch.js-2.x)
+- [React InstantSearch Hooks](https://codesandbox.io/s/github/algolia/instantsearch/tree/templates/react-instantsearch-hooks)
```
Comment (FabienMotte) [inline packages/create-instantsearch-app/README.md:155]: Same as my previous comment.

**Review by dhayab — APPROVED** [review]

**Review by Haroenv — APPROVED** [review]

File: packages/create-instantsearch-app/src/templates/InstantSearch.js 2/index.html.hbs [hunk 2]
```diff
@@ -24,7 +24,7 @@
     </h1>
     <p class="header-subtitle">
       using
-      <a href="https://github.com/algolia/instantsearch.js">
+      <a href="https://github.com/algolia/instantsearch/tree/master/packages/instantsearch.js">
```
[the remark this answers is not in the document]
Comment (FabienMotte) [reply packages/create-instantsearch-app/src/templates/InstantSearch.js 2/index.html.hbs:27]: Is this one correct? https://github.com/algolia/instantsearch/tree/v2.10.5

[the remark this answers is not in the document]
Comment (Haroenv) [reply packages/create-instantsearch-app/src/templates/InstantSearch.js 2/index.html.hbs:27]: yes, that would be correct, good catch

[the remark this answers is not in the document]
Comment (FabienMotte) [reply packages/create-instantsearch-app/src/templates/InstantSearch.js 2/index.html.hbs]: Thanks for your suggestion!

**Pull request merged**